shell/browser/api/message_port.cc

```diff
@@ -126,8 +125,7 @@ void MessagePort::Entangle(blink::MessagePortDescriptor port) {
   v8::Isolate* isolate = v8::Isolate::GetCurrent();
   v8::HandleScope scope(isolate);
   connector_ = std::make_unique<mojo::Connector>(
-      port_.TakeHandleToEntangle(
-          blink::ExecutionContext::From(isolate->GetCurrentContext())),
+      port_.TakeHandleToEntangleWithEmbedder(),
```

Comment:
Oh nice—how did you find this? Did you just know to look for it?

Reply:
@deepak1556 ☝️

Reply:
I knew our message port implementation was based on `third_party/blink/renderer/core/messaging/message_port.cc` , I was looking at the changes in that CL corresponding to the file, saw there are two new apis to take the handle. Secondly, since our message port api operates on the main process too, I wanted to eliminate the usage of `blink::ExecutionContext`, fortunately the CL had a use case similar. Thereafter it was just looking at the implementations of the new apis and the only difference happened to be instrumentation call, so adopted it safely.